Documentation.md

```diff
@@ -0,0 +1,207 @@
+
+The following commands are available globally.\
+Modmail for a specific user can be enabled again with `;enableModmail <user>`. After modmail get enabled for a user, he will l only get the notification about when he can contact again once.  Modmail can also be disabled for a specific user via `;disableModmail <user>`. If a staff member wants to contact a user directly, without waiting for the user to write `;contact <user>` serves this purpose. In case there is already a thread going on, an embed will be posted containing a link.
+
+Current implementation only supports one image attachment per message.
+    
+### Ban
+
+`;ban @Username#1234 reason`
+
+Ban user from server, reason parameter is optional.\
+Command can also be used with [user IDs](https://dis.gd/userid) to ban users not present in server with `;banid` command.\
+It is made so that other bots can't be banned with it.
+
+### Kick
+
+`;kick @Username#1234 reason`
+
+Kick user from server, reason parameter is optional.\
+Command can also be used with [user IDs](https://dis.gd/userid)
+It is made so that other bots can't be kicked with it.
+
+### Mute
+
+Mute users in both text and voice chats. It will give them the roles "Voice muted" and "Text muted" (requires prior setup in database).\
+Syntax is `;mute @Username#1234 duration reason`
+Possible durations are days (d), hours (h), minutes (m) and seconds (s).\
+Reason field is optional.\
+Eg `;mute @Username#1234 5d2h1m3s Get muted` to mute user Username#1234 for 5 days, 2 hours, 1 minute and 3 seconds.\
+Command can also be used with [user IDs](https://dis.gd/userid)
+It is made so that other bots can't be muted with it.
+
+### Bulk delete
+
+Self explanatory, use `;purge` followed by number of messages you want to prune.\
+Currently limited to 100 messages.
+
+### Warnings
+
+Allow moderators (users with **Staff** role) to warn users.\
+A warning can be given by using command `;warn @Username#1234 reason` (reason field is optional). Also works with [user IDs](https://dis.gd/userid).\
+When someone is warned their warning is named as "active": three active warnings will lead to a server ban (as of writing banning command has to be issued by a moderator).\
+Every 90 days (delay configurable in database) warnings are decayed so that we can actually keep track of persons infractions.\
+The check for that action happens everyday at 00:00 UTC.
+
+Warnings list can be queried by moderators by issuing `;warnings` command. If list takes more than one embed it is possible to navigate using the arrows reaction. Warning of a specific user can be queried by using command `;warnings @Username#1234` which also works with [user IDs](https://dis.gd/userid). Clicking on wastebasket reaction will delete the embed, otherwise the embed will delete by itself after two minutes without interaction with embed by user that requested list of warnings.
```

Comment:
note that active and decayed warnings are shown, Active warnings have a text 'Active'